docs/modules/introduction_main.rst

```diff
@@ -1,9 +1,35 @@
 
-Introduction
-============
+==============================================
+ Welcome to the PythonRobotic Documentation 👋
+==============================================
+
+Welcome to PythonRobotic's comprehensive documentation, an open source collection of tried-and-tested robotics algorithms designed to simplify and expedite your robotics journey. 
```

Comment:
I'm sorry for the confusion, but I'd like this Introduction chapter to be a general introduction to robotics, not specifically to this OSS. However, I think this text is excellent, so could you please move it to the 'Getting started' page? https://atsushisakai.github.io/PythonRobotics/getting_started.html"

Reply:
Ok , I'm moving this text to the “Getting started' page”.

Reply:
Can I describe a little bit about OSS in the introduction section? 
Like many users do not know what exactly OSS means. It would be good to add a brief description of OSS in the introduction section.